nav2_docking/opennav_docking/src/docking_server.cpp

```diff
@@ -385,6 +391,17 @@ void DockingServer::doInitialPerception(Dock * dock, geometry_msgs::msg::PoseSta
   publishDockingFeedback(DockRobot::Feedback::INITIAL_PERCEPTION);
   rclcpp::Rate loop_rate(controller_frequency_);
   auto start = this->now();
+  // If docking without sensors, stop the robot for short time to get a stable and quality detection of docking pose
+  if(dock_backwards_without_sensor_){
```

Comment:
This shouldn't be necessary. If we're at the initial staging pose, we're already stopped.